src/tests/lib/components/Input.svelte.spec.ts

```diff
@@ -517,6 +518,32 @@ describe("Input", () => {
       expect(input.value).toBe("0.000000011231232121");
     });
 
+    it("should not trim below given decimals", async () => {
```

Comment:
Can you add `0.999999999999999876` to the tests since you mentionned it as comment and I used it for having a look?